app/components/Modals/TokenModal/utils.js

```diff
@@ -8,14 +8,28 @@ export const getNewTokenItem = (networkId: string) => ({
   isUserGenerated: true
 })
 
+const ensureHex = (token: string): boolean => {
+  const hexRegex = /^([0-9A-Fa-f]{2})*$/
```

Comment:
Its better to move the `hexRegex` declaration out of the function (no need to recreate it every time, just reuse it)

Reply:
👍 will definitely move this to a util file (I meant to do that but was moving quickly) thanks